_sass/components/_dropdown.scss

```diff
@@ -1,188 +1,11 @@
-$dropdown-min-width: 125px;
-
-.dropdown-arrow {
-  display: none;
-}
-
 @include at-media("mobile-lg") {
   .dropdown {
-    position: relative;
-    z-index: 1;
-
     li {
```

Comment:
Some of this is still used. Most of the rest of the styles are implemented elsewhere:

https://github.com/18F/identity-site/blob/9597157ae42cc2da04f55cd36bff76bd1d0eb114/_sass/components/_nav.scss#L23-L44